app/models/saved_claim.rb

```diff
@@ -18,6 +18,9 @@
 class SavedClaim < ApplicationRecord
   include SetGuid
 
+  # Default zip code for Lighthouse Benefits Intake API when address is unavailable
+  DEFAULT_ZIP_CODE = '00000'
+
```

Comment:
How did you come up with this default? I have fuzzy memories that this zip is used to decide which regional processing center processes the claim.